rl_games/common/experience.py

```diff
@@ -194,3 +195,98 @@ def update_priorities(self, idxes, priorities):
 
             self._max_priority = max(self._max_priority, priority)
 
+
+class VectorizedReplayBuffer:
```

Comment:
I  think you can use pytorch tensors everywhere instead of numpy.
Here are bonuses:
1) you can put everything on gpu
2) no need to cast to the torch tensor